pydrive2/test/README.rst

```diff
@@ -25,11 +27,13 @@ Run tests locally
       tests", and click **Create**. Select **Continue** at the next **Service
       account permissions** page, click at **+ CREATE KEY**, select **P12** and
       **Create**. Save generated **.p12** key file at your local disk.
-    - Edit **/pydrive2/test/settings/local/default.yaml** by replacing email
-      **your-service-account-email** with your new created service account
+    - Edit files **/pydrive2/test/settings/local/default.yaml** and
+      **/pydrive2/test/settings/local/test_oauth_test_06.yaml** by replacing
+      **your-service-account-email** with email of your new created service account
       and **your-file-path.p12** with path to the downloaded **.p12** key file.
+      Value for key **client_user_email** should be left blank.
 
-3. Create an OAuth client ID and configure tests with it:
+4. Create an OAuth client ID and configure tests with it:
```

Comment:
in this point - is it a requirement to create configs in the root?

Reply:
PyDrive settings .yaml files reference files as `client_config_file: 'configs/client_secrets.json'`
The path is relative to the current working directory.

Reply:
so we can probably change this?